src/GitVersionCore/GitVersionComputer.cs

```diff
@@ -10,86 +8,62 @@
 
 namespace GitVersion
 {
-    public class ExecuteCore
+    public class GitVersionComputer : IGitVersionComputer
```

Comment:
Do you think `Computer` or `Calculator` is a better word for what this class does?

Reply:
`GitVersionCalculator` sounds better, have changed the methods as well: `CalculateVersionVariables` and `TryCalculateVersionVariables`